apps/server/src/modules/tool/controller/tool-configuration.controller.ts

```diff
@@ -22,26 +22,47 @@ export class ToolConfigurationController {
 		private readonly externalToolResponseMapper: ExternalToolResponseMapper
 	) {}
 
-	@Get('available/:scope/:id')
+	@Get('available/school/:id')
 	@ApiForbiddenResponse()
-	async getAvailableToolsForSchool(
+	public async getAvailableToolsForSchool(
 		@CurrentUser() currentUser: ICurrentUser,
-		@Param() scopeParams: ScopeParams,
 		@Param() idParams: IdParams
 	): Promise<ToolConfigurationListResponse> {
 		const availableTools: ExternalToolDO[] = await this.externalToolConfigurationUc.getAvailableToolsForSchool(
 			currentUser.userId,
 			idParams.id
 		);
+
 		const mapped: ToolConfigurationListResponse =
 			this.externalToolResponseMapper.mapExternalToolDOsToToolConfigurationListResponse(availableTools);
+
+		return mapped;
+	}
+
+	@Get('available/:context/:id')
+	@ApiForbiddenResponse()
```

Comment:
I think we should document our api with more in informations. I know we dont do that at the moment but we should do. Could you some more information e.g. like:
  
  @Get()
  @ApiBearerAuth()
  @ApiUnauthorized()
  @ApiOperation({ summary: 'a summary' })
  @ApiOkResponse({
    description: 'some description',
    type: ResponeObject,
  })